addon/components/ember-flatpickr.js

```diff
@@ -16,28 +19,13 @@ import TextField from '@ember/component/text-field';
  *
  * @class EmberFlatpickr
  * @element EmberFlatpickr
- * @extends TextField
  * @public
  * @uses Flatpickr
  */
-export default TextField.extend(Flatpickr, {
-  /**
-   * ARIA bindings for a textbox.
-   * @see https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/textbox_role
-   * @see https://labs.levelaccess.com/index.php/Category:ARIA
-   */
-  attributeBindings: [
-    'aria-activedescendent',
```

Comment:
We should probably mention in the docs how to add some of these back. With `...attributes` I think they can just be passed directly, but wouldn't hurt to have an example.